test/k8s-integration/driver-config.go

```diff
@@ -38,10 +40,41 @@ func generateDriverConfigFile(pkgDir, storageClassFile string) (string, error) {
 	defer w.Flush()
 
 	// Fill in template parameters
+	caps := []string{
+		"persistence",
+		"block",
+		"fsGroup",
+		"exec",
+		"multipods",
+		"topology",
+	}
```

Comment:
Is there a well known source of truth for all valid capability values? Is it part of the CSI capability spec?

Reply:
They're here - definitely not easy to find.
https://github.com/kubernetes/kubernetes/blob/23320c958edc85feb541db85edc43ec629288a7d/test/e2e/storage/testsuites/testdriver.go#L136

Reply:
Add a comment where to find the capabilities from?